ext/gd/tests/gdimage_cloning.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+Checks that GdImage instances can be cloned from gd > 2.2.0
+--SKIPIF--
+<?php if (version_compare(GD_VERSION, '2.2.0', '<')) { die("Skipped: GdImage is not clonable before gd 2.2.0"); } ?>
```

Comment:
Not absoultely sure, but I think that would result in "reason: ped: GdImage …". I also would just drop the braces.
```suggestion
<?php if (version_compare(GD_VERSION, '2.2.0', '<')) die("skip GdImage is not clonable before gd 2.2.0"); } ?>
```

Reply:
(You forgot to remove the trailing `}`)

Reply:
Ah, right, thank you! Fixed now.